site/gatsby-site/src/components/reports/ReportCard.js

```diff
@@ -55,11 +54,11 @@ const ReportCard = ({ item, className = '', incidentId }) => {
       >
         <div className="flex items-center w-full justify-between">
           <div>
-            <LocalizedLink to={`#r${item.report_number}`} className="max-w-full cursor-pointer">
+            <a href={`#r${item.report_number}`} className="max-w-full cursor-pointer">
```

Comment:
`LocalizedLink` doesn't handle the hash?

Reply:
It adds a trailing slash `/cite/1/#r1/` which breaks the scrolling.